Chiazokam/Politico
# #163351854 create the edit party endpoint

#### What does this PR do?
This PR tests and creates the edit party endpoint

#### Description of Task to be completed?
- write tests for all the functions
- create the endpoint

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-datastructures-edit-party-163351854`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Make a `PATCH` request to `http://localhost:3000/api/v1/parties/1/name `
- You should receive a successful response

#### Any background context you want to provide?
- the POST endpoint payload should be structured like this- 
`
{
      "partyName": "People Congress"
}
`
#### What are the relevant pivotal tracker stories?
[#163351854](https://www.pivotaltracker.com/story/show/163351854)

####  Images

![image](https://user-images.githubusercontent.com/26940294/51784724-c3a3c880-214d-11e9-8092-2c31cf6972fe.png)
